src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -279,7 +279,8 @@ public void validateAppEngineJavaComponents()
       throws AppEngineJavaComponentsNotInstalledException {
     if (!Files.isDirectory(getJavaAppEngineSdkPath())) {
       throw new AppEngineJavaComponentsNotInstalledException(
-          "Validation Error: Java App Engine components not installed");
+          "Validation Error: Java App Engine components not installed."
+              + " Fix by running 'gcloud components install app-engine-java' on command-line.");
```

Comment:
LGTM

On Tue, Sep 13, 2016 at 10:55 AM, meltsufin <notifications@github.com>
wrote:

> In src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java
> <https://github.com/GoogleCloudPlatform/appengine-plugins-core/pull/209#discussion_r78576307>
> :
>
> > @@ -279,7 +279,8 @@ public void validateAppEngineJavaComponents()
> >        throws AppEngineJavaComponentsNotInstalledException {
> >      if (!Files.isDirectory(getJavaAppEngineSdkPath())) {
> >        throw new AppEngineJavaComponentsNotInstalledException(
> > -          "Validation Error: Java App Engine components not installed");
> > +          "Validation Error: Java App Engine components not installed."
> > +              + " Fix by running 'gcloud components install app-engine-java' on command-line.");
>
> I would revisit this when these exceptions are made checked. For the time
> being, however, since the clients are not forced to handle the exception,
> I'd rather have this in there to potentially save time for users
> encountering this issue.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/appengine-plugins-core/pull/209/files/812c9fe0ad1de7606eba5cf57cb2666074e46082..9f25195bd2eec34cd90b4ea84afef00a70f3384e#r78576307>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHf5HcaQD45YP_kpZZsYkHvhOZFIx9HXks5qprlMgaJpZM4J68gI>
> .
>